B2C-Tfp-Issuer.md

```diff
@@ -0,0 +1,14 @@
+# Azure AD B2C issuer claim
+
+Azure AD B2C user flows can be customized to emit issuer claim in two formats: `https://<domain>/<tenant>/v2.0/` and `https://<domain>/tfp/<tenant>/<userFlow>/v2.0/`. Originally `tfp` (which stands for Trust Framework Policy) in the URI was used to differentiate between Azure AD and Azure AD B2C . Currently the B2C team is working to streamline the experience and remove `tfp`. Therefore, the current guidance from Azure B2C team is to use `https://<domain>/<tenant>/v2.0/` format. 
```

Comment:
i don't think you need to explain this. Just mention we support the default value.

Reply:
@jennyf19 
I updated this wiki entry.